python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -293,8 +301,13 @@ def runAssociation(self, sensorRef):
         self.log.info("Running Association...")
 
         diffType = self.config.differencer.coaddName
+        psfType = self.config.differencer.measurement.slots.psfFlux
         try:
             catalog = sensorRef.get(diffType + "Diff_diaSrc")
+            # TODO: workaround for DM-14507
```

Comment:
Now that we've found the bug in ap_association that was causing this do you want to wait for me to merge that ticket or do this for now and then remove it later?

Reply:
Sure, I can wait.

Reply:
This is not fixed in DM-14517. Thanks for the help.